.obelisk/impl/github.json

```diff
@@ -2,6 +2,6 @@
   "owner": "obsidiansystems",
   "repo": "obelisk",
   "branch": "srid-https-deploy--upstream-acme",
```

Comment:
Shouldn't this point to `reunification` branch?

Reply:
thanks, missed that